packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -42,7 +42,12 @@ class ExpansionTile extends StatefulWidget {
     this.trailing,
     this.initiallyExpanded = false,
     this.tilePadding,
+    this.expandedCrossAxisAlignment,
+    this.expandedAlignment,
   }) : assert(initiallyExpanded != null),
+       assert(expandedCrossAxisAlignment != CrossAxisAlignment.baseline,
+       'CrossAxisAlignment.baseline is not supported since the expanded children '
+           'are aligned in a column, not a row. Try to use another constant.'),
```

Comment:
nit: I find this format easier to read, but this one is up to you:
```suggestion
       assert(
         expandedCrossAxisAlignment != CrossAxisAlignment.baseline,
         'CrossAxisAlignment.baseline is not supported since the expanded children '
         'are aligned in a column, not a row. Try to use another constant.',
       ),
```

Reply:
I was trying to save vertical space, but I'll do the changes, it will be easier to read.